help/audacity.appdata.xml

```diff
@@ -3,12 +3,12 @@
   <developer_name>Muse Group</developer_name>
   <id>org.audacityteam.Audacity</id>
   <launchable type="desktop-id">audacity.desktop</launchable>
-  <project_license>GPL-2.0</project_license>
+  <project_license>GPL-3.0</project_license>
```

Comment:
Note that the appdata license field must use an SPDX identifier according to its spec, and this identifier was deprecated in SPDX version 3.0. It is replaced with either `GPL-3.0-only` or `GPL-3.0-or-later` depending on which you mean. If this field parses to something that isn't a valid SPDX identifier, then it can cause programs to think the app is proprietary.

Reply:
@LWinterberg can you please check?

Reply:
@imciner2, well spotted! The plain GPL SPDX identifiers were indeed deprecated as they were [considered ambiguous](https://www.gnu.org/licenses/identify-licenses-clearly.html). We need to use `GPL-3.0-only` due to the inclusion of VST3, even though most of our code is compatible with `GPL-2.0-or-later`. @LWinterberg, sorry I missed this in my review.